README.md

```diff
@@ -50,29 +50,35 @@ In order to run Toriel locally, you'll need to [join the Hack Club Slack](https:
      SLACK_SIGNING_SECRET=SIGNING_SECRET
      SLACK_BOT_TOKEN=Bot_User_OAuth_Token
      ```
-     where `SIGNING_SECRET` is your app's Signing Secret which you can find by clicking on "Basic Information" in the settings bar and scrolling down (click show to and copy it)
+     where `SIGNING_SECRET` is your app's Signing Secret, which you can find by clicking on "Basic Information" in the settings bar and scrolling down (click show to and copy it)
          ![Screenshot of where to click show and copy the signing secret](https://cloud-j9zzknpea-hack-club-bot.vercel.app/0screenshot_2022-04-18_at_6.49.53_pm.png)
      and `Bot_User_OAuth_Token` is found under "OAuth & Permissions" (You will need click "Install to Workspace" before you can view the token)
      ![Screenshot of where the token is](https://cloud-twxncowk1-hack-club-bot.vercel.app/0screenshot_2022-04-18_at_7.00.44_pm.png)
 12. [Create a private channel](https://slack.com/help/articles/201402297-Create-a-channel) where your app can run the welcome flow. Similar to the role that #in-the-cave plays for Toriel (run `toriel-restart` in slack to see what this means).
      - Invite your bot to that channel (you can @mention it to add)
 13. Edit [transcript.yml](/util/transcript.yml)
      - The key:value pairs under `channels:` represent `channel-name:channel-id` and these are referred to elsewhere in the codebase with `{channels.channel-name}`
+     
      ![Screenshot of channel list](https://cloud-5prq93r05-hack-club-bot.vercel.app/0screenshot_2022-04-18_at_9.12.10_pm.png)
-     - You can get the channel id by opening the channel and clicking on the channel name in the top left. Scroll down the modal and you should see it (it's in the bottom left)
-     - Hence, you should add your own bot to these channels **or** you can delete channels you can comment out channels you don't need for now with `#` and just add your bot to channels you do want it to join. 
-     - Also, you want to add your own private channel here. ex. if your channel is `msw-test-cave` with the channel id `CDF1A5EG865` you would add 
+     - You can add your own bot to these channels for testing **or** you can delete or comment out channels you don't need for testing
```

Comment:
This part confused me slightly - I feel like I'm missing something in my rewording about adding your bot to channels you want it in.

On the same note, is the `IMPORTANT` caveat in the right spot here, as I moved it?

Reply:
hey jessica, oo thank you! this looks perfect! hmm, the way is phrased sounds good to be but essentially the idea is that you need to make sure your bot is in all the channels listed (and if you don't want to add it, you could also delete it but you might break something so be careful). if your bot doesn't have access to these channels when they call it later in the flow, the bot will break. does this make sense?

and the `IMPORTANT` caveat is great! 

thank you sm :))

Reply:
@bellesea Awesome! That makes a lot of sense, thank you :)